code/script.js

```diff
@@ -226,7 +229,7 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-  
+  console.log(secret)
```

Comment:
Remember to remove the console.log, otherwise, player can open the inspector and find out the secret person 😉